src/rsz/src/RepairSetup.cc

```diff
@@ -1588,4 +1632,201 @@ void RepairSetup::printProgress(const int iteration,
   }
 }
 
+// Terminate progress if incremental fix rate within an opto interval falls
+// below the threshold.   Bump up the threshold after each large opto internval.
```

Comment:
```suggestion
// below the threshold.   Bump up the threshold after each large opto interval.
```

Reply:
Fixed.   We need a spell checker for comments.